apps/charterafrica/src/payload/blocks/Spotlight.js

```diff
@@ -0,0 +1,120 @@
+import link from "../fields/link";
+
+const Spotlight = {
+  slug: "spotlight",
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      label: {
+        en: "Title",
+        fr: "Titre",
+        pt: "Título",
+      },
+      required: true,
+      localized: true,
+    },
+    {
+      name: "items",
+      type: "array",
+      minRows: 1,
```

Comment:
```suggestion
      minRows: 1,
      maxRows: 4,
```

Don't think the design can handle more than 4.